### PR TITLE
Lingua_franca addition

### PR DESCRIPTION
**Pull request by mllopartbsc**
Addition of lingua franca to the TTS API.

### CODE REVIEW

File: server/server.py [hunk 1]
```diff
@@ -318,6 +328,29 @@ async def details(request: Request):
...
+            return sentence
+
+        sentence = re.sub(r'les\s+', '', sentence, count=1)
+
+        # Iterate through matches and substitute with formatted time string
+        for match in matches:
+            H = int(match[0])
+            M = int(match[1])
+            dt = datetime.datetime(2017, 1, 31, H, M, 0, tzinfo=default_timezone())  # Using UTC timezone for simplicity
+            formatted_time = nice_time(dt, lang="ca")  # Assuming you have a function to format time in Catalan
```
Comment (gullabi) [inline server/server.py]: For the chosen behaviour this line should be `formatted_time = nice_time(dt, lang="ca", use_24hour=True)`

File: Dockerfile [hunk 2]
```diff
@@ -22,6 +22,8 @@ RUN pip install --upgrade pip && \
  make && \
  make install
 
+RUN git clone https://github.com/MycroftAI/lingua-franca.git
```
Comment (gullabi) [inline Dockerfile]: shouldn't we use one of the forks? Carme and Joan had fixes in their forks. (probably does not affect the hour formatting)

File: Dockerfile
Comment (gullabi) [inline Dockerfile:1]: the new `espeak-ng` seem to have pronunciation problems.

File: server/server.py (hunk 1 above)
Comment (mllopartbsc) [reply server/server.py]: Should be resolved in the latest commit.

File: server/server.py [hunk 3]
```diff
@@ -318,6 +328,29 @@ async def details(request: Request):
 
 
 def worker(sentence, speaker_id, model, use_aliases, new_speaker_ids):
+    def substitute_time(sentence):
+        # Regular expression to find time pattern (HH:MM)
+        time_pattern = re.compile(r'(\d{1,2}):(\d{2})')
```
Comment (gullabi) [inline server/server.py]: throws value errors in the case 154:00 or 15:001, correct regex should be `((?<=\s)\d{1,2}):(\d{2}(?=\s))`

**Pull request merged**